release-notes/6.0/api-diff/.Net/6.0.0_System.Collections.Immutable.md

```diff
@@ -0,0 +1,9 @@
+# System.Collections.Immutable
+
+``` diff
+ namespace System.Collections.Immutable {
+-    public struct ImmutableArray<T> : ICollection, ICollection<T>, IEnumerable, IEnumerable<T>, IEquatable<ImmutableArray<T>>, IImmutableList<T>, IList, IList<T>, IReadOnlyCollection<T>, IReadOnlyList<T>, IStructuralComparable, IStructuralEquatable
++    public readonly struct ImmutableArray<T> : ICollection, ICollection<T>, IEnumerable, IEnumerable<T>, IEquatable<ImmutableArray<T>>, IImmutableList<T>, IList, IList<T>, IReadOnlyCollection<T>, IReadOnlyList<T>, IStructuralComparable, IStructuralEquatable
```

Comment:
Ah I see, the only difference here was adding the readonly keyword was this api diff on this file intended or cant we just state that we just added ``readonly`` to this struct (and probably a few other things perhaps)?